envs/postgres-s3-external-spark-otel/docker-compose.yml

```diff
@@ -0,0 +1,176 @@
+    ports:
+      - 1888:1888 # pprof extension
+      - 8888:8888 # Prometheus metrics exposed by the Collector
+      - 8889:8889 # Prometheus exporter metrics
+      - 13133:13133 # health_check extension
+      - 4317:4317 # OTLP gRPC receiver
+      - 4318:4318 # OTLP http receiver
+      - 55679:55679 # zpages extension
+    volumes:
+      - ${DAG_FOLDER}/otel-config/otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml
+    links:
+      - tempo
+
+  postgresdb:
+    image: postgres:17
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_HOST=postgresdb
+      - POSTGRES_PORT
+      - POSTGRES_PASSWORD
+      - POSTGRES_USER
+      - POSTGRES_DB
+
+  sparkmaster:
+    build:
+      context: ${DOCKER_CONTEXT_FOLDER}/aws-spark
+      dockerfile: Dockerfile
+      args:
+        - SPARK_VERSION=${SPARK_VERSION}
+    environment:
+      - SPARK_NO_DAEMONIZE=true
+      - SPARK_MASTER_HOST=0.0.0.0
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SERVER
+      - AWS_PORT
+      - OTEL_TRACES_SAMPLER
+      - OTEL_TRACES_EXPORTER
+      - OTEL_EXPORTER_OTLP_ENDPOINT
+    ports:
+      - 7077:7077
+      - 18080:8080
+    entrypoint:
+      - /usr/spark/sbin/start-master.sh
+    links:
+      - s3server:${DEMO_BUCKET}.s3server
+
+  sparkworker:
+    build:
+      context: ${DOCKER_CONTEXT_FOLDER}/aws-spark
+      dockerfile: Dockerfile
+      args:
+        - SPARK_VERSION=${SPARK_VERSION}
+    environment:
+      - SPARK_NO_DAEMONIZE=true
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SERVER
+      - AWS_PORT
+      - OTEL_TRACES_SAMPLER
+      - OTEL_TRACES_EXPORTER
+      - OTEL_EXPORTER_OTLP_ENDPOINT
+    ports:
+      - 18081:8081
+    entrypoint:
+      - /usr/spark/sbin/start-slave.sh
+      - spark://sparkmaster:7077
+      - "-m"
+      - "8G"
+    depends_on:
+      - sparkmaster
+    links:
+      - s3server:${DEMO_BUCKET}.s3server
+
+  # Grafana, Tempo taken from https://github.com/grafana/tempo/blob/60780f7ff878d86e0e23179fc65f2c3ed9832e9d/example/docker-compose/otel-collector/docker-compose.yaml
+  tempo-init:
+    image: &tempoImage grafana/tempo:latest
+    user: root
+    entrypoint:
+      - "chown"
+      - "10001:10001"
+      - "/var/tempo"
+    volumes:
+      - ${DAG_FOLDER}/_tempo-data:/var/tempo
+
+  tempo:
+    image: *tempoImage
+    command: [ "-config.file=/etc/tempo.yaml" ]
+    volumes:
+      - ${DAG_FOLDER}/grafana/tempo.yml:/etc/tempo.yaml
+      - ${DAG_FOLDER}/_tempo-data:/var/tempo
+    ports:
+      - "3200"   # tempo
+      - "4317"  # otlp grpc
+      - "4318"  # otlp http
+    depends_on:
+      - tempo-init
+
+  prometheus:
+    image: prom/prometheus:latest
+    command:
+      - --config.file=/etc/prometheus.yaml
+      - --web.enable-remote-write-receiver
+      - --enable-feature=exemplar-storage
+      - --enable-feature=native-histograms
+    volumes:
+      - ../shared/prometheus.yml:/etc/prometheus.yaml
```

Comment:
I don't see this config file. All seems to be working but I guess with some default prometheus conf instead of a dedicated one